VimeoNetworking/VimeoNetworking/VimeoClient.swift

```diff
@@ -279,6 +279,11 @@ final public class VimeoClient
         return RequestToken(task: requestTask)
     }
     
+    public func removeCachedResponse<ModelType: MappableResponse>(request request: Request<ModelType>)
```

Comment:
This took me a minute to unpack, but I'm reading this as, "remove the cached response of a generic model type, which is any `Request` object that conforms to `MappableResponse`." Does that sound right?

Sometimes I think Swift is not the easiest to read.